Kickstarter-iOS/Features/DesignSystem/Colors/DesignSystemBackgroundColors.swift

```diff
@@ -0,0 +1,95 @@
+import UIKit
+
+public enum DesignSystemBackgroundColors {
+  case backgroundSurfacePrimary
+  case backgroundSurfaceInverse
+  case backgroundDisabled
+  case backgroundInverse
+  case backgroundInverseHover
+  case backgroundInversePressed
+  case backgroundSelected
+  case backgroundAction
+  case backgroundActionHover
+  case backgroundActionDisabled
+  case backgroundActionPressed
+  case backgroundAccentGreenBold
+  case backgroundAccentGreenSubtle
+  case backgroundAccentBlueBold
+  case backgroundAccentBlueSubtle
+  case backgroundAccentPurpleSubtle
+  case backgroundDangerBold
+  case backgroundDangerSubtle
+  case backgroundDangerBoldPressed
+  case backgroundDangerBoldHovered
+  case backgroundDangerSubtleHovered
+  case backgroundAccentGrayBold
+  case backgroundAccentGraySubtle
+  case backgroundWarningBold
+  case backgroundWarningSubtle
+
+  var rawValue: String {
+    switch self {
+    case .backgroundSurfacePrimary:
+      return DesignSystemCoreColors.white.rawValue
+    case .backgroundSurfaceInverse:
+      return DesignSystemCoreColors.grey10.rawValue
+    case .backgroundDisabled:
+      return DesignSystemCoreColors.grey4.rawValue
+    case .backgroundInverse:
+      return DesignSystemCoreColors.white.rawValue
+    case .backgroundInverseHover:
```

Comment:
Do we want to support hover colors on mobile? What situation would we use them in?

Reply:
see my comment above for more context. This will likely change as needed.